source/functions/initializeWecSim.m

```diff
@@ -273,8 +281,8 @@
 if ~isempty(idx)
     for kk = 1:length(idx)
         it = idx(kk);
-        body(it).hydroForcePre(waves.omega,waves.direction,simu.cicTime,waves.bem.count,simu.dt,...
-            simu.rho,simu.gravity,waves.type,waves.waveAmpTime,simu.stateSpace,simu.b2b);
+        body(it).hydroForcePre(waves(1).omega,waves(1).direction,simu.cicTime,waves(1).bem.count,simu.dt,...
```

Comment:
@salhus I want to document discussion had on the previous PR to make sure they are captured upon merge. No action needed and placed a project board card to revisit this discussion in FY24.

"Moderate comment: Is there a reason that waves(1) is hard coded to the first wave instance? This seems to indicate that you cannot mix and match certain wave features. This relates to my request on documentation to state what combinations of sea states are possible.

Yes that is correct. I think this can be expanded and I am planning on enabling mix-n-match in next FY.
I will update documentation to reflect this limitation."